aos/image_ingestion/ingest_phosim_image.ipynb

```diff
@@ -0,0 +1,341 @@
+      "text/plain": [
+       "<Figure size 720x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "dataRoot=isrDir.name\n",
+    "repoRoot = os.path.join(dataRoot,) \n",
+    "butler = dafPersist.Butler(repoRoot, )\n",
+    "raw_image = {}\n",
+    "for sensor in ['S11','S10']:\n",
+    "    dataId = dict(run='9006001', visit=4021123106001, raftName='R22', detectorName=sensor)\n",
+    "    exposure = butler.get('raw', **dataId) \n",
+    "    image = exposure.image.array\n",
+    "    raw_image[sensor]=image\n",
+    "    \n",
+    "fig,ax = plt.subplots(1,2,figsize=(10,5))\n",
+    "\n",
+    "i = 0 \n",
+    "for sensor in raw_image.keys():\n",
+    "    ax[i].imshow(raw_image[sensor], origin='lower', vmax=1200)\n",
+    "    ax[i].set_title(sensor)\n",
+    "    i += 1 \n",
+    "fig.suptitle('Raw images', fontsize=18)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Do the ISR:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# do the ISR \n",
+    "isrWrapper.config(doFlat=True, doOverscan=True)\n",
+    "isrWrapper.doISR(isrDir.name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Show the post-ISR images: "
+   ]
+  },
+  {
```

Comment:
Can you add a color bar here and show that we are actually doing a good job normalizing the different amps and this is not just an artifact of color scaling to the bright donuts?

Reply:
Added a colorbar